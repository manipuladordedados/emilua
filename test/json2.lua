local json = require('json')

local x = json.decode('{"hello": "world"}')
print(x.hello)

local heavily_nested_json = json.decode('{"a":{"b":{"c":{"d":{"e":{"f":{"g":{"h":{"i":{"j":{"k":{"l":{"m":{"n":{"o":{"p":{"q":{"r":{"s":{"t":{"u":{"v":{"w":{"x":{"y":{"z1":{},"z2":{}}}}}}}}}}}}}}}}}}}}}}}}}}}')
print(heavily_nested_json.a.b.c.d.e.f.g.h.i.j.k.l.m.n.o.p.q.r.s.t.u.v.w.x.y.z1)
print(heavily_nested_json.a.b.c.d.e.f.g.h.i.j.k.l.m.n.o.p.q.r.s.t.u.v.w.x.y.z2)
