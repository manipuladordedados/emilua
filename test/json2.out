world
table: 0x1
table: 0x2
